style.css

```diff
@@ -1,58 +1,72 @@
 body {
     background-color: white;
+    font-family: 'Open Sans', sans-serif;
+    width: 1500px;
```

Comment:
Instead of giving width to the body, create a new div inside and set its width.
Make sure the CV is centered horizontally.

![image](https://user-images.githubusercontent.com/15181215/187093250-3ba50beb-8fba-4fbd-884f-60b47550a00b.png)